polling_stations/apps/data_finder/views.py

```diff
@@ -103,29 +103,49 @@ class BasePollingStationView(TemplateView, LogLookUpMixin):
 
 class PostcodeView(BasePollingStationView):
 
-    def get_polling_station(self, location):
+    def get_polling_station(self, location, council_id):
         try:
             polling_district = PollingDistrict.objects.get(
                 area__covers=location)
         except PollingDistrict.DoesNotExist:
             return None
 
+        if not polling_district:
+            # is this different to PollingDistrict.DoesNotExist exception?
+            return None
```

Comment:
I think this will never be hit, as per the comment.  `.get` returns either the object, or raises a `DoesNotExist` exception.